adapters/wagmi-connector/src/bloctoConnector.ts

```diff
@@ -64,7 +65,7 @@ class BloctoConnector extends Connector<
     return provider
       .request({ method: 'eth_requestAccounts' })
       .then(
-        (accounts: string[]): `0x${string}` => accounts[0] as `0x${string}`
+        (accounts): `0x${string}` => (accounts as string)[0] as `0x${string}`
```

Comment:
Thanks for notice array, but should be
```ts
.then((accounts): `0x${string}` => (accounts as `0x${string}`[])[0]);
```
because type 'unknown' is not assignable to type '`0x${string}`[]'. Can only do type assertion here.

Reply:
Oh I think it's my VSCode cache too.

Then could you update `EthereumProviderInterface` in `packages/blocto-sdk/src/providers/types/ethereum.d.ts` like
```js
export interface EthereumProviderInterface extends BloctoProviderInterface,  IEthereumProvider {
    ...
  request<T>(args: EIP1193RequestPayload): Promise<T>;
  ...
}
```

And then we can use it in `getAccount` like
```ts
return provider
      .request<`0x${string}`[]>({ method: 'eth_requestAccounts' })
      .then((accounts): `0x${string}` => accounts[0]);
```